c10/util/intrusive_ptr.h

```diff
@@ -369,7 +369,7 @@ class intrusive_ptr final {
   }
 
   intrusive_ptr& operator=(intrusive_ptr&& rhs) & noexcept {
-    return operator=<TTarget, NullType>(std::move(rhs));
+    return operator= <TTarget, NullType>(std::move(rhs));
```

Comment:
@Skylion007 do you know why it does that my any chance?

Reply:
A rule to format operator= with template initialisation?